databasin/client.py

```diff
@@ -149,7 +150,51 @@ def get_temporary_file(self, uuid):
         except HTTPException as e:
             raise_for_authorization(e.response, self.username is not None)
             raise
+    
+    def import_lpk(self, lpk_file):
+        if lpk_file.endswith('.lpk'):
+            f = open(lpk_file, 'a+b')
+        else:
+            raise ValueError('File must be an ArcGIS Layer Package with a .lpk extension')
+
+        filename = '{0}.lpk'.format(os.path.splitext(os.path.basename(lpk_file))[0])
+        
+        tmp_file = self.upload_temporary_file(f, filename=filename)
+
+        f.close()
 
+        tmp_job_args = {
+            'file': tmp_file.uuid,
+            'url': None,
+            'dataset_type': 'ArcGIS_Native'
+        }
+       
+        tmp_job = self.create_job('create_import_job', job_args=tmp_job_args, block=True)
```

Comment:
`tmp_job` is a misnomer. It's really an "import" job, but simply `job` would be fine, too.